packages/react/ds/src/primitives/anchor.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+type AnchorProps = React.AnchorHTMLAttributes<HTMLAnchorElement> & {
+  external?: boolean;
```

Comment:
There is a Link component with ```external``` property, maybe adding a variant there instead of creating a new anchor component?

Reply:
Hey @ramonmulia thank you for the feedback.

The `Link` already have an external prop, but it also contains the `gi-link` style class by default.
That's why I created a primitive one with no style.

Reply:
Maybe I can use Anchor primitive component inside the link component to avoid duplicated code.
Leaving only the logic inside the primitive
What you think?

Reply:
Good catch, I like this idea